scintilla/lexers/LexUser.cxx

```diff
@@ -624,6 +624,25 @@ static bool IsNumber(StyleContext & sc, vector<string> * numberTokens[], vvstrin
     }
 }
 
+static inline void AddSubGroupSpecialCharacter(const char * s, vector<string> & subvector)
+{
+    if (!strcmp(s, "EOL"))
+    {
+        subvector.push_back("\r\n");
+        subvector.push_back("\n");
+        subvector.push_back("\r");
+    }
+    else if (!strcmp(s, "WSP"))
+    {
+        subvector.push_back("\s");

```

Comment:
Is `\s` interpreted as a ASCII space or as a Regular Expression white-space?
If it's the latter `\t` is already part of `\s` i.e. Regular Expression `\s` matches a space, a tab, a line break, or a form feed.